pro/src/screens/SignupJourneyForm/Offerers/Offerers.tsx

```diff
@@ -93,7 +93,7 @@ const Offerers = (): JSX.Element => {
                   displayToggleVenueList && !isVenueListOpen && index >= 4
                 }
               >
-                {venue.name}
+                {venue?.publicName ?? venue.name}
```

Comment:
Pas besoin du premier `?`, non ?
```suggestion
                {venue.publicName ?? venue.name}
```

Reply:
Bien vu, merci :)